File: Listas de Exercícios (Python)/Python/Lista 2 - Sala/10.py
import time

print("Mês (1 a 12)\n")

M=int(input("Mês (forma numérica [1 a 12]): "))

print("")

if (M==1):
    print("Janeiro")
elif (M==2):
    print("Fevereiro")
elif (M==3):
    print("Março")
elif (M==4):
    print("Abril")
elif (M==5):
    print("Maio")
elif (M==6):
    print("Junho")
elif (M==7):
    print("Julho")
elif (M==8):
    print("Agosto")
elif (M==9):
    print("Setembro")
elif (M==10):
    print("Outubro")
elif (M==11):
    print("Novembro")
elif (M==12):
    print("Dezembro")
else:
    print("Digite um valor entre 1 e 12!!")

print("")

time.sleep(3)

File: Lista de Exercícios (C)/C/Lista 2 - Sala/10.c
#include <stdio.h>

int main()
{
    int M;
    
    printf("Mês (1 a 12)\n\n");
    
    printf("Mês (forma numérica [1 a 12]): "); scanf("%d", &M);
    
    printf("\n");
    
    if (M==1) {
        printf("Janeiro");
    }
    else if (M==2) {
        printf("Fevereiro");
    }
    else if (M==3) {
        printf("Março");
    }
    else if (M==4) {
        printf("Abril");
    }
    else if (M==5) {
        printf("Maio");
    }
    else if (M==6) {
        printf("Junho");
    }
    else if (M==7) {
        printf("Julho");
    }
    else if (M==8) {
        printf("Agosto");
    }
    else if (M==9) {
        printf("Setembro");
    }
    else if (M==10) {
        printf("Outubro");
    }
    else if (M==11) {
        printf("Novembro");
    }
    else if (M==12) {
        printf("Dezembro");
    }
    else {
        printf("Digite um valor entre 1 e 12!!");
    }
}